### ISSUE TITLE
Scheduled task collector not showing in prometheus after added in windows exporter

### ISSUE THREAD

**Issue by Siva-22m** [prometheus-community/windows_exporter#1286]
I've installed Prometheus version 2.6.0 and Windows exporter version 0.6 in my prod server. I need to monitor the task scheduler event details. I'm using a batch file to run the windows exporter msi file and I've added the respective **Scheduled_task** collector in that.

**msiexec /i C:\Users\Administrator\Downloads\windows_exporter.msi ENABLED_COLLECTORS="ad,iis,logon,memory,process,tcp,thermalzone" TEXTFILE_DIR="C:\custom metrics\"** 

By using this command 

But after restarting both the windows exporter and prometheus services, I can't able to views the newly added collector in the metrics drop down list or localhost:9090/metrics page.

**Comment by DiniFarb** [comment]
Well first of all in your `ENABLED_COLLECTORS` is no `scheduled_task` listed so it won't monitor task scheduler at all. And you might need to consider filtering if your machine has a lot of tasks your not interested in. See the according docs for the [scheduled_task](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.scheduled_task.md) module. 

Secondly I think version v0.6.0 did not have the `scheduled_task` so I suggest you to use a newer or better the latest version v0.23.1.

And lastly I suggest you to check the event viewer log if you can't see anything in Prometheus, maybe the exporter is logging some errors.

**Comment by Siva-22m** [comment]
Thank you for your response @DiniFarb, so may I know from what version of windows exporter supports `Scheduled_task`. And by the way the `ENABLED_COLLECTORS` which I mentioned here is the piece of command which I copy pasted from the official documentation and not from my config file.

**Comment by DiniFarb** [comment]
I looked trough the releases, it was added in [v0.19.0](https://github.com/prometheus-community/windows_exporter/releases/tag/v0.19.0) but there were bugs discovered later and fixed with [v0.21.0](https://github.com/prometheus-community/windows_exporter/releases/tag/v0.21.0) 

The installation docs are in the main [readme](https://github.com/prometheus-community/windows_exporter#installation) besides of that you can refer to the [msiexec](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec) docs.

But I think you can just uninstall the service with:
```
msiexec /x <path-to-old-msi-file>
```
and then reinstall with the new msi file and your extra parameters like:
```
msiexec /i <path-to-new-msi-file> ENABLED_COLLECTORS="ad,iis,logon,memory,process,tcp,thermalzone,scheduled_task" TEXTFILE_DIR="C:\custom metrics"
```

**Comment by Siva-22m** [comment]
Thank you sooo much buddy @DiniFarb. I'll look in to the version of supported windows exporter
